pkg/email/template.go

```diff
@@ -0,0 +1,65 @@
+package email
+
+import (
+	_ "embed"
+	"html/template"
+	"net/url"
+	"path"
+	"strings"
+)
+
+const (
+	ResetPasswordURLPath = "/auth/resetpassword" //#nosec
+	InviteUserURLPath    = "/auth/users/create"  //#nosec
+
+	resetPasswordEmailSubject = "Reset Password Request for your lakeFS account"
+	inviteUserWEmailSubject   = "You have been invited to lakeFS"
+)
+
+var (
+	//go:embed invite_user_template.html
+	invite string
```

Comment:
If you write each embed follow by the template, you probably get a better text alignment